app/components/admin/users-table.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Table,
+  Tr,
+  Th,
+  Thead,
+  Tbody,
+  TableContainer,
+  Td,
+  Card,
+  IconButton,
+  Tooltip,
+  useToast,
+  Flex,
+} from '@chakra-ui/react';
+import { CopyIcon, DeleteIcon } from '@chakra-ui/icons';
+import { FaTheaterMasks } from 'react-icons/fa';
+
+import type { User } from '@prisma/client';
+
+interface UsersTableProps {
+  users: User[];
+}
+
+export default function UsersTable({ users }: UsersTableProps) {
+  const toast = useToast();
+
+  function onCopyNameToClipboard(subdomain: string) {
+    navigator.clipboard.writeText(subdomain);
+    toast({
+      title: 'Email was copied to clipboard',
+      position: 'bottom-right',
+      status: 'success',
+    });
+  }
+
+  return (
+    <Card p="2" mt="4">
+      <TableContainer>
+        <Table variant="striped" colorScheme="gray">
+          <Thead>
+            <Tr>
+              <Th>Email</Th>
+              <Th>Name</Th>
+              <Th>Records</Th>
+              <Th>Certificate status</Th>
+              <Th>Role</Th>
+              <Th />
+            </Tr>
+          </Thead>
+          <Tbody>
+            {users.map((user) => {
+              return (
+                <Tr key={user.email}>
+                  <Td>
+                    <Flex justifyContent="space-between" alignItems="center">
+                      {user.email}
+                      <Tooltip label="Copy email to clipboard">
+                        <IconButton
+                          icon={<CopyIcon color="black" boxSize="5" />}
+                          variant="ghost"
+                          ml="2"
+                          onClick={() => onCopyNameToClipboard(user.email)}
+                          aria-label="copy email"
+                        />
+                      </Tooltip>
+                    </Flex>
+                  </Td>
+                  <Td>{user.displayName}</Td>
+                  <Td>0</Td>
+                  <Td>Status</Td>
+                  <Td>{user.group}</Td>
```

Comment:
I think we should skip showing this column, it will make things easier.

Reply:
Could you please explain what is the problem with this one?